### PR TITLE
convert to markdown and sync with english version for `WheelEvent`

### PR DESCRIPTION
**Pull request by yin1999**

### CODE REVIEW

File: files/zh-cn/web/api/wheelevent/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{{InheritanceDiagram}}
+
+## 构造函数
+
+- {{domxref("WheelEvent.WheelEvent", "WheelEvent()")}}
+  - : 创建一个 `WheelEvent` 对象。
+
+## 属性
+
+_该接口继承了父接口 {{domxref("MouseEvent")}}、{{domxref("UIEvent")}} 和 {{domxref("Event")}} 的属性。_
```
Comment (awxiaoxian2020) [inline files/zh-cn/web/api/wheelevent/index.md:29]:
```suggestion
*该接口继承了父接口 {{domxref("MouseEvent")}}、{{domxref("UIEvent")}} 和 {{domxref("Event")}} 的属性。*
```

File: files/zh-cn/web/api/wheelevent/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- {{DOMxRef("WheelEvent.wheelDeltaX")}} {{ReadOnlyInline}} {{deprecated_inline}}
+  - : 返回一个整数，表示水平滚动量。
+- {{DOMxRef("WheelEvent.wheelDeltaY")}} {{ReadOnlyInline}} {{deprecated_inline}}
+  - : 返回一个整数，表示垂直滚动量。
+
+> **备注：** [元素：mousewheel 事件](/zh-CN/docs/Web/API/Element/mousewheel_event)有关于已弃用的属性（`wheelDelta`、`wheelDeltaX` 和 `wheelDeltaY`）的额外说明。
+
+## 方法
+
+_该接口本身未定义方法，但继承了父接口 {{domxref("MouseEvent")}}、{{domxref("UIEvent")}}、{{domxref("Event")}} 的方法。_
```
Comment (awxiaoxian2020) [inline files/zh-cn/web/api/wheelevent/index.md:57]:
```suggestion
*该接口本身未定义方法，但继承了父接口 {{domxref("MouseEvent")}}、{{domxref("UIEvent")}}、{{domxref("Event")}} 的方法。*
```

File: files/zh-cn/web/api/wheelevent/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/wheelevent/index.md:29]: It's ok here, the difference between `_` and `*` is:

```markdown
hello_world_
hello _world_

hello*world*
hello *world*
```

rendered as:

```html
<p>hello_world_</p>
<p>hello <em>world</em></p>

<p>hello<em>world</em></p>
<p>hello <em>world</em></p>
```

**Review by awxiaoxian2020 — APPROVED** [review]

File: files/zh-cn/web/api/wheelevent/index.md (hunk 1 above)
Comment (awxiaoxian2020) [reply files/zh-cn/web/api/wheelevent/index.md:29]: Thanks.

**Pull request merged**